src/dc_receive_imf.rs

```diff
@@ -1640,6 +1674,11 @@ async fn check_verified_properties(
 
     ensure!(mimeparser.was_encrypted(), "This message is not encrypted.");
 
+    ensure!(
+        mimeparser.get(HeaderDef::ChatVerified).is_some(),
```

Comment:
Good that it is checked, but maybe it should be applied? If `ChatVerified` is set to the wrong value, this probably means we have lost the message changing the group state.

Reply:
might also be the sender has lost the message, so i would not degrade to unprotected here. 

for group creation, however, we already use this value (eg. in case the group was deleted for whatever reason).

Reply:
Yes, I mean not downgrading, but upgrading. I commented on this line of code because it checks the header, but for upgrading it should be checked somewhere else.